datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/metadata.py

```diff
@@ -481,20 +481,22 @@ def metadata(check, check_duplicates, show_warnings):
                     (echo_failure, f"{current_check}:{line} interval should be an int, found '{row['interval']}'.")
                 )
 
+            # check if curated_metric is valid
+            if row['curated_metric']:
```

Comment:
I didn't include the same logic as before here since the validation can verify that the number of columns is correct. If `curated_metric` is included as a column on the top, then it will be validated.